content/docs/community/automated-testing.md

```diff
@@ -5,46 +5,35 @@ description: 'Our automated testing strategy and resources'
-
-
-## ChatOps Usage
-
-To run the tests, you'll need to issue some commands by posting GitHub comments on the PR. You can use any of the following to perform tests or run certain actions:
+In order to securely run terraform tests and save on infrastructure costs we've setup some basic "ChatOps" style functionality. To use it you will need to have at least `triage` level of access to a repo. Typically, this is done by a CloudPosse team upon PR review.
 
+Tests are initiated by posting GitHub comments on the PR. Currently supported commands are the following:
 
 | Command                                             | Description                                                                        |
 |-----------------------------------------------------|------------------------------------------------------------------------------------|
-| `/test bats`                                        | Run bats automated tests                                                           |
-| `/test readme`                                      | Verify the `README.md` is current and updated from `README.yaml`                   |
-| `/test terratest`                                   | Run the `terratest` integration tests in `test/src`                                |
-| `/test all`                                         | Run `readme`, `bats`, and `terratest` tests                                        |
-| `/test bats readme`, `/test readme terratest`, etc. | Run any combination of `/test` commands from one comment.                          |
-| `/rebuild-readme`                                   | Run `make init && make readme` on the current code and commit it back to the repo. |
-| `/terraform-fmt`                                    | Run `terraform fmt` on the current code and commit it back to the repo.            |
+| `/terratest`                                        | Run the `terratest` integration tests in `test/src`                                |
 
 
-All tests run from the [`cloudposse/actions`](https://github.com/cloudposse/actions/actions) repository.
+Terraform tests run against our [testing infrastructure](https://github.com/cloudposse/testing.cloudposse.co) that we host in an isolated account on AWS, strictly for the purposes of testing infrastructure.
 
+ChatOps is powered by [GitHub Actions](https://github.com/features/actions) and the [slash-dispatch-command](https://github.com/peter-evans/slash-command-dispatch).
+
+The terratest workflow is defined in the [`cloudposse/actions`](https://github.com/cloudposse/actions/blob/master/.github/workflows/terratest-command.yml) repository. The benefit with this is that we have one place to control the testing
+workflow for all of our hundreds of terraform modules. The downside, however, with dispatched workflows is that the _workflows_ always run from the `master` branch.
```

Comment:
We don't talk about the workflows, but should. Discuss the shared workflows and the general flow.

Reply:
Sufficient that someone on the team could understand the various pieces and pick it up.

Reply:
It's a missing section here, I will add. 
The one above should stay as is until we get rid of dipatching terratest workflows to `cloudposse/action`

Reply:
@osterman Added shared workflow wording to the very top section